src/biogeochem/SatellitePhenologyMod.F90

```diff
@@ -24,7 +24,8 @@ module SatellitePhenologyMod
   private
   !
   ! !PUBLIC MEMBER FUNCTIONS:
-  public :: SatellitePhenology     ! CLMSP Ecosystem dynamics: phenology, vegetation
+  public :: GetSatellitePhenologyInputs ! put the data into the correct format
+  public :: SetSatellitePhenologyBGCCanopyStructs ! CLM(BGC)-SP phenology and vegetation
```

Comment:
I'm just wondering about changing the name to

SetSatellitePhenologyBGCCanopyTypes

Reply:
sure sounds good

Reply:
Compsets: Sp, Bgc, BgcCrop, Fates, FatesSp